### ISSUE TITLE
centrifugo V4 admin interface not compatible with reverse proxy

### ISSUE THREAD

**Issue by maheus** [centrifugal/web#59]
Hi,

centrifugo admin panel seems not compatible with reverse proxy .
example:
reverse proxy send request  `/toto` to centrifugo on `/`
if i send `/toto/admin`, i receive a responce but the JS seems hardcoded with full path 
```
<script defer="defer" src="/static/js/main.850cfbc8.js"></script><link href="/static/css/main.0c407e63.css" rel="stylesheet">
```
so the js are not found on my server

but on v3, the paths are relative and it works
```
<script type="text/javascript" src="bundle.js?7c53f4001f2f014e1a63"></script>
```

**Comment by FZambia** [comment]
Hello @maheus

Thanks for the detailed report, missed this difference. I'll try to fix it in the next Centrifugo release.

**Comment by FZambia** [comment]
I believe this should be fixed by https://github.com/centrifugal/web/commit/e24a4ee18fb7f7667f8831f4b32d08778678e2f9, the master branch of Centrifugo also updated to inherit the fix in embedded web UI: https://github.com/centrifugal/centrifugo/commit/ac7796198923df6496ca378016a6015dbc4dab52

**Comment by maheus** [comment]
thanks

**Comment by FZambia** [comment]
Welcome! Let's keep this issue open till the release.

**Comment by FZambia** [comment]
Released in v4.1.1, @maheus please try it out and reopen in case of any issues.
